java/lance-jni/src/jni_helpers.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 Lance Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use jni::objects::{JMap, JObject, JString};
+use jni::JNIEnv;
+
+pub fn throw_java_exception(env: &mut JNIEnv, err_msg: &str) {
+    env.throw_new("java/lang/RuntimeException", err_msg)
```

Comment:
nit: should we make a `com.lancedb.lance.LanceException` class? a bare `RuntimeException` might make catch statements unnecessarily wide

Reply:
ok, will do as follow up.